pages/api/submit.tsx

```diff
@@ -181,4 +183,28 @@ const processFormData = async (
   }
 };
 
+/**
+ * Push a given file to a temporairy S3
+ * @param fileOrArray
+ * @param reqFields
+ * @param key
+ */
+const pushFileToS3 = async (
+  fileOrArray: formidable.File,
+  fileName: string
+): Promise<UploadResult> => {
+  const bucketName: string = process.env.AWS_BUCKET_NAME as string;
+  let uploadResult: UploadResult;
+  try {
+    uploadResult = await uploadFileToS3(fileOrArray, bucketName, fileName);
+    if (!uploadResult.isValid) {
+      throw new Error(uploadResult.errorReason);
+    }
+  } catch (error) {
+    throw new Error(error);
+  } finally {
+    fs.unlinkSync(fileOrArray.path);
```

Comment:
This is good for now.  We can open a new issue later that defines more enhanced error handing for when to delete files from S3 if there is an error later in the calling function.